src/screens/explore/About/BlendedAbout/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { BackHandler } from 'react-native';
+import About from '../../About';
+
+interface BlendedAboutProps {
+  route: { params: { course } },
+  navigation: {
+    navigate: (path: string, params?: object) => {},
+  },
+}
+
+const BlendedAbout = ({ route, navigation }: BlendedAboutProps) => {
+  const { course } = route.params;
+  const { program } = course.subProgram;
```

Comment:
`course` vient du back du coup il faut peut etre verifier que subProgram existe bien sinon ici on risque d'avoir une erreur de type 'cant get program of undefined'

Reply:
done